espwrap/adaptors/sendgrid_v3.py

```diff
@@ -174,6 +182,29 @@ def message_constructor(self, to_send):
 
         return message
 
+    def generate_subject(self, email, name):
+        """
+        Generate email subject for recipient.
+        If username contains "." or "+" it is considered an alias so we
+        prepend a custom greeting to avoid bounces from providers like Gmail
+        :param email: Recipient email
+        :param name: Recipient name
+        :return subject: sendgrid.helpers.mail.Subject
+        """
+        try:
```

Comment:
What if, instead of custom greeting, we just appended `[email+adddress@something.com]`

So it would be 

`Some subject` for jim@spoton.com
and
`Some subject [jim+1@spoton.com]` for jim+1@spoton.com

Does that make sense? That way we dont have to worry about localization

Reply:
Sounds good. I did some tests and it worked too.